examples/platformer/main.go

```diff
@@ -72,8 +72,10 @@ const (
 )
 
 type char struct {
-	x  int
-	y  int
+	//Positions
```

Comment:
Follow the Go comment convention. Moreover, I don't think we need this comment as this is too obvious.